package.json

```diff
@@ -39,19 +39,19 @@
   "peerDependencies": {
     "@a-ui/core": "^3.0.3",
     "@a-ui/flexboxgrid": "^1.0.0",
-    "@angular-redux/store": "~9.0.0",
-    "@angular/animations": "^6.1.0",
-    "@angular/common": "^6.1.0",
-    "@angular/core": "^6.1.0",
-    "@angular/forms": "^6.1.0",
-    "@angular/http": "^6.1.0",
-    "@angular/language-service": "^6.1.0",
-    "@angular/platform-browser": "^6.1.0",
-    "@angular/platform-browser-dynamic": "^6.1.0",
-    "@angular/router": "^6.1.0",
-    "redux": "~4.0.1",
-    "rxjs": "~6.1.0",
-    "typescript": "~2.9.2"
+    "@angular-redux/store": ">9.0.0",
```

Comment:
is er een reden dat dit `>` is en niet `>=`?

Reply:
Good catch